packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -903,12 +950,14 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
             left: 0.0,
             right: 0.0,
             bottom: bottom,
-            child: widget.underline ?? Container(
-              height: 1.0,
-              decoration: const BoxDecoration(
-                border: Border(bottom: BorderSide(color: Color(0xFFBDBDBD), width: 0.0))
-              ),
-            ),
+            child: widget.underline ??
```

Comment:
Formatting for this block needs to be fixed

Reply:
Done